demo/allegro5/Makefile

```diff
@@ -1,8 +1,8 @@
 # Install
 BIN = demo
 
-# Flags
-CFLAGS += -std=c99 -pedantic -O2
+PARENT_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST)))/..)
```

Comment:
Have you tested this on *nix (including macOS) platforms as well as Windows (MSVC)? I'm afraid this won't work well on all of them (or maybe I'm too pessimistic :wink:).